chapters/publication.tex

```diff
@@ -1,96 +1,171 @@
-They allow more advanced integration with Git,
-among other advantages, but the entire team needs to be comfortable
+who simply want to get to writing, like senior researchers.
+While integrated editing and compiling tools like TeXStudio\sidenote{
+  \url{https://www.texstudio.org}}
+and \texttt{atom-latex}\sidenote{
+  \url{https://atom.io/packages/atom-latex}}
+offer the most flexibility to work with \LaTeX\ on your computer,
+such as advanced integration with Git,
+the entire group of writers needs to be comfortable
 with \LaTeX\ before adopting one of these tools.
+They can require a lot of troubleshooting at a basic level at first,
+and non-technical staff may not be willing or able to acquire the required knowledge.
+Therefore, to take advantage of the features of \LaTeX,
+while making it easy and accessible to the entire writing team,
+we need to abstract away from the technical details where possible.
 
-One of the important tools available in \LaTeX\ is the \textbf{BibTeX bibliography manager}.\sidenote{\url{http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.365.3194&rep=rep1&type=pdf}}
-This tool stores unformatted references
-in an accompanying \texttt{.bib} file,
-and \LaTeX\ then inserts them in text
-using the \texttt{\textbackslash cite\{\}} command.
-With this structure, \LaTeX\ automatically pulls
-all the citations into text. \LaTeX\ allows you to specify
-how they should be displayed in text
-(ie, as superscripts, inline references, etc.)
-and how the bibliography should be styled and in what order.
-A reference in the \texttt{.bib} file
-can be copied directly from Google Scholar
-by clicking "BibTeX" at the bottom of the Cite window.
+One of the most important tools available in \LaTeX\ is the BibTeX bibliography manager.\sidenote{
+  \url{http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.365.3194&rep=rep1&type=pdf}}
+BibTeX keeps all the references you might use in an auxiliary file,
+then references them as plain text in the document using a \LaTeX\ command.
+The same principles that apply to figures and tables are therefore applied here:
+You can make changes to the references in one place (the \texttt{.bib} file),
+and then everywhere they are used they are updated correctly with one process.
+Specifically, \LaTeX\ inserts references in text using the \texttt{\textbackslash cite\{\}} command.
+Once this is written, \LaTeX\ automatically pulls all the citations into text
+and creates a complete bibliography based on the citations you use when you compile the document.
+The system allows you to specify exactly how references should be displayed in text
+(such as superscripts, inline references, etc.)
+as well as how the bibliography should be styled and in what order
+(such as Chicago, MLA, Harvard, or other common styles).
+To obtain the references for the \texttt{.bib} file,
+you can copy the specification directly from Google Scholar
+by clicking ``BibTeX'' at the bottom of the Cite window.
```

Comment:
That it is integrated in Google Scholar is something that I think we can use as an argument for how wide spread and standard LaTeX is in our field. And another argument for why LaTeX should be the first tool we mention in this chapter

Reply:
59c0b527ba910f600326cde8bbec40677612b93e